cloud-nio/cloud-nio-impl-drs/src/main/scala/cloud/nio/impl/drs/DrsPathResolver.scala

```diff
@@ -120,7 +120,15 @@ object MarthaResponseSupport {
   def convertMarthaResponseV2ToV3(response: MarthaV2Response): MarthaResponse = {
     val dataObject = response.dos.data_object
     val size = dataObject.size
-    val timeUpdated = dataObject.updated.map(OffsetDateTime.parse(_).toString)
+
+    val timeUpdated: Option[String] = dataObject.updated map { time: String =>
+      if (time.endsWith("Z")) {
+        OffsetDateTime.parse(time).toString
+      } else {
+        LocalDateTime.parse(time).toString
+      }
+    }
```

Comment:
If there is a more sophisticated existing library function that would do this all in one go, that would be great